Packs/Rapid7AppSec/Integrations/Rapid7AppSec/Rapid7AppSec_description.md

```diff
@@ -0,0 +1,17 @@
+### Basic Information
+Supports AppSec 3.0.0 and above.
+
+When using XSOAR credentials insert your API key to the credential password. Otherwise, insert your API key into the relevant required parameters.
```

Comment:
```suggestion
When using Cortex XSOAR credentials, insert your API key as the credential password. Otherwise, insert your API key into the relevant required parameters.
```